opta/core/cloud_client.py

```diff
@@ -41,3 +41,7 @@ def cluster_exist(self) -> bool:
     @abstractmethod
     def set_kube_config(self) -> None:
         raise NotImplementedError()
+
+    @abstractmethod
+    def get_kubectl_context_name(self) -> str:
+        pass
```

Comment:
why not raise NotImplementedError() like others?

Reply:
it actually does if you try to run